### PR TITLE
Add sample code for decoding string and binary payloads

### PR DESCRIPTION
**Pull request by TechPreacher**
- Add ByteArrayToString helper method
- Add example decoders for string and binary payloads
- Remove unnecessary using statements

### CODE REVIEW

File: Samples/DecoderSample/Classes/LoraDecoders.cs [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-using System.Threading.Tasks;
+﻿using System.Text;
 using Newtonsoft.Json;
 
 namespace SensorDecoderModule.Classes
 {
     internal static class LoraDecoders
     {   
-        private static string DecoderValueSensor(byte[] payload, uint fport)
+        private static string DecoderStringValueSensor(byte[] payload, uint fport)
```
Comment (ronniesa) [inline Samples/DecoderSample/Classes/LoraDecoders.cs]: It is a bit tricky that we have in the NtwSrv the name DecoderValueSensor and here the same with another name. Also if you change it here the doc should reflect it. I would keep it DecoderValueSensor if not we need to change docs and the NtwSrv

Comment (TechPreacher) [reply Samples/DecoderSample/Classes/LoraDecoders.cs]: Good input. I keep one DecoderValueSensor and use code & comments inside to show how to decode both type of payloads...!

Comment (TechPreacher) [reply Samples/DecoderSample/Classes/LoraDecoders.cs]: Updated!

**Comment by TechPreacher** [comment]
@ronniesa what do you think? Leave all in DecoderValueSensor or follow @fbeltrao 's input to make DecoderValueSensor and DecoderBinaryValueSensor? Lots of discussion on a small change. Sorry for that!

**Comment by ronniesa** [comment]
I would say keep it like this if not we need to document more etc..,..

**Pull request merged**